cobalt/BUILD.gn

```diff
@@ -13,6 +13,53 @@
 # limitations under the License.
 
 group("gn_all") {
+  testonly = true
+
   # TODO(b/371589344): Fix android build configs.
-  deps = [ "//starboard($starboard_toolchain)" ]
+  deps = [
+    ":cobalt",
+    "//starboard($starboard_toolchain)",
+  ]
+}
+
+if (is_android) {
+  import("//build/config/android/config.gni")
```

Comment:
Why is this import needed?

Reply:
Most of this is taken I left that in from content/shell/BUILD.gn to keep it a little closer for Android to work too. I can strip the entire `if (is_android)` is that is more clear.

Reply:
Deleted. PR update follows